script/cibuild

```diff
@@ -6,7 +6,7 @@ set -e
 # place with this version at the beginning of each test and many commands have
 # conditional logic based on the remote version. Running the suite against
 # different major versions ensures we're covering these conditional paths.
-REMOTE_VERSIONS="2.18.0 2.20.0"
+REMOTE_VERSIONS="2.18.0 2.19.0 2.20.0"
```

Comment:
do we need to add 2.21.0 as well since it is releasing right now

Reply:
No. I am going to wait for the release to go in and then update my addition to be 2.20.

Reply:
>different major versions

maybe we could call it `different release versions`

Reply:
ok, may I know where is the logic we have version validation?

Reply:
This has been removed in the latest iteration of the PR, so I think we're good here.